src/DataHandling/XMLExtractor.php

```diff
@@ -0,0 +1,148 @@
+<?php
+namespace CubeTools\CubeCommonBundle\DataHandling;
+
+use Symfony\Component\DomCrawler\Crawler;
+
+class XMLExtractor
+{
+    /**
+     * @var \Symfony\Component\DomCrawler\Crawler
+     */
+    protected $crawler;
+    
```

Comment:
ah, committing in eclipse does not run the hooks. Right, this is why I did not recommend anybody to use this. Have forgotten about. Can we simply work around this by adding it to eclipse somehow?